script.js

```diff
@@ -1,4 +1,285 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+// not my proudest work but this is what i have been able to do in my limited time.
+
+/*Some global variables*/
+// Create an array to store the cards that the player has  (player's hand)
+var playerCards = [];
+// Create an array to store the computer's cards.  (computer's hand)
+var computerCards = [];
+
+var gameMode = 0;
```

Comment:
Would recommend against using numbers as game modes- makes it more difficult to understand what is going on! better to use more descriptive game modes in English.

One thing you can do is to define game modes as `CONSTANT_VALUES` in uppercase snake case. 

E.g. 
```
var START_OF_GAME = 0;
var HIT_OR_STAND = 1;
var END_OF_GAME = 2;
var gameMode = START_OF_GAME;
```

Then in your `main` function when you are checking for game modes, you can compare `gameMode` with any of the constant value game modes you defined for better readability.